contracts/contracts/vault/VaultStorage.sol

```diff
@@ -250,6 +250,11 @@ contract VaultStorage is Initializable, Governable {
     // For future use
     uint256[44] private __gap;
 
+    constructor() Governable() {
```

Comment:
We could not call `Governable()` here as well. Since it will only set the governor to `msg.sender` and then have it overwritten by the 0 governor.